src/cockpit/389-console/src/lib/plugins/memberOf.jsx

```diff
@@ -1638,10 +1644,9 @@ class MemberOf extends React.Component {
                             <GridItem className="ds-left-margin" span={3}>
                                 <Button
                                     variant="primary"
-                                    isDisabled={memberOfConfigEntry === "" || !valid_dn(memberOfConfigEntry)}
```

Comment:
I think we should still validate the DN here. It'll allow to avoid situation, when we already have a `Shared Config Entry`, but then the user enters an invalid `Shared Config Entry` DN and then presses `Manage` button.

But the case probably should be like this (`!==` instead of `===`):

```js
isDisabled={(memberOfConfigEntry !== "" && !valid_dn(memberOfConfigEntry))}
```

Reply:
Heh, whoops :-)

Reply:
Actually I made the text field to always be disabled - forcing the user to use the modal to create, edit, and delete the shared config entry.  I think it's more intuitive and removes the chance to use it incorrectly.

Reply:
What I mean, if you first Create the entry, and then back on the main screen will try to modify the `Shared Config Entry` it will allow to enter Invalid DN

Reply:
I don't allow the user to enter/change a value on the main page anymore.

Reply:
> I don't allow the user to enter/change a value on the main page anymore.

Sounds good.
I'm a bit concerned that it'll make the logic in other `Shared Config Entry` different which is a bit confusing. Should we then change all of the  `Shared Config Entry`  field logic? I think it's only `Account Policy` plugin, though.

Another thing, if we always disable the field, then it doesn't make sense to have it as an input field as it's less readable. Probably, it can be changed to `Text`? I don't have a hard opinion here, though.

Reply:
> > I don't allow the user to enter/change a value on the main page anymore.
> 
> Sounds good. I'm a bit concerned that it'll make the logic in other `Shared Config Entry` different which is a bit confusing. Should we then change all of the `Shared Config Entry` field logic? I think it's only `Account Policy` plugin, though.

Yes we should make then all the same, and it's an easy change, so I'll do that quick.

> 
> Another thing, if we always disable the field, then it doesn't make sense to have it as an input field as it's less readable. Probably, it can be changed to `Text`? I don't have a hard opinion here, though.

Yeah this crossed my mind, I thought the "disabled input" was done in other areas for read-only values but it was changed at point.  So I'll redo this as well.

Reply:
@droideck ok made all three plugins (account policy, MO, and RI) all behave and look thd same now.  Please review